docs/feel-expression.md

```diff
@@ -175,7 +175,7 @@ true or null
 // true
 
 false or null  
-// false
```

Comment:
Did this change because of your change? Or was this wrong in the documentation?

Reply:
Apparently I can only comment near to lines where you made changes.

This comment is about places where you didn't make changes:

- "Evaluate a Unary Tests" -> "Evaluate a Unary Tests" / "Evalaute Unary Tests"
- Below is a section on external functions. This section would benefit from a hint that external functions are disabled by default

Reply:
Again. The documentation was a lie.

I added the comment in the other PR: https://github.com/camunda/feel-scala/pull/116/files#diff-eea0eae9141e7419ee431be85fc00127R341